File: app/src/main/java/com/androidnerds/doordash/data/remote/feed/IStoreFeedRemoteDataSource.java
package com.androidnerds.doordash.data.remote.feed;

import com.androidnerds.doordash.data.remote.model.StoreFeedDTO;

import io.reactivex.rxjava3.core.Observable;
import io.reactivex.rxjava3.core.Single;


public interface IStoreFeedRemoteDataSource {

    Single<StoreFeedDTO> getStoreFeed(double latitude, double longitude, int limit, long offset);
}

File: app/src/main/java/com/androidnerds/doordash/storefeed/data/remote/IStoreFeedRemoteDataSource.kt
package com.androidnerds.doordash.storefeed.data.remote

import com.androidnerds.doordash.storefeed.domain.StoreFeed
import io.reactivex.Flowable

interface IStoreFeedRemoteDataSource {

    fun getStoreFeed(location: Location, limit: Int, offset: Int) : Flowable<StoreFeed>
}